### DISCUSSION TITLE
3T$ market cap milestone

### DISCUSSION THREAD

**Discussion by diegoglozano** [has2k1/plotnine#835] [2024 Plotnine Contest]
### Authors

Diego Garcia Lozano

### Links

[LinkedIn](https://www.linkedin.com/in/diegoglozano/)

### Full description

As you may know, only 3 companies have reached the 3T$ of market cap milestone: Apple, Microsoft and NVIDIA. But, how much time did they take to reach that milestone? Let’s check it using polars and plotnine!

[Substack post](https://diegoglozano.substack.com/p/2024-plotnine-contest)

### Code repository

https://github.com/diegoglozano/plotnine-contest/tree/main

### Result
![6_plot](https://github.com/user-attachments/assets/dfd1b2a6-9869-46db-a6be-b3e6f674d2b6)

**Comment by has2k1** [comment]
![3T](https://github.com/user-attachments/assets/d449369b-b74c-4ae0-a381-de9a63bd586c)

**Comment by EconomiCurtis** [comment]
Ahoy @diegoglozano , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@diegoglozano, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
